app/src/main/java/com/example/datarsd1/forecastapp/Utils.java

```diff
@@ -0,0 +1,76 @@
+package com.example.datarsd1.forecastapp;
+
+import android.content.Context;
+import android.content.Intent;
+import android.location.Address;
+import android.location.Geocoder;
+import android.os.AsyncTask;
+
+import org.json.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Adhiraj Datar on 3/6/2017.
+ */
+
+public class Utils
+{
+
+    Context context;
+
+    public Utils(Context context)
+    {
+        this.context = context;
+    }
```

Comment:
cleaner to pass in geocoder, no need to have constructor to pass in context and can keep all methods static